antora-playbook.yml

```diff
@@ -29,12 +29,12 @@ content:
     - url: .
       branches: HEAD
       start_path: home
-    # - url: https://git@github.com/couchbasecloud/couchbase-cloud
-    # branches: [AV-22069_Capella-GA-Docs]
-    # start_path: docs/public
-    # - url: https://git@github.com/couchbase/couchbase-operator
-    # branches: [2.3.0-beta, 2.2.x, 2.1.x, 2.0.x, 1.2.x, 1.1.x, 1.0.x]
-    # start_path: docs/user
+    - url: https://git@github.com/couchbasecloud/couchbase-cloud
```

Comment:
Looks like this cleanup is mostly enabling a whole lot of projects?
Sure that Capella and Mobile etc. are hot topics, but do we need all the other things (Talend etc.) below?
Just thinking about build times etc.

Reply:
This was available in og `sandbox` as well so I've not removed: https://github.com/couchbase/docs-site/blob/sandbox/antora-playbook.yml#L43

We can trim the fat later, but currently just trying to get this in a publishable state first.